libgraph/test/property-view-fixed-binary.cpp

```diff
@@ -0,0 +1,70 @@
+#include "katana/Properties.h"
+#include "katana/Result.h"
+#include "katana/analytics/Utils.h"
+#include "tsuba/TxnContext.h"
+
+namespace {
+
+constexpr int kNumRows = 10;
+constexpr int kNumArrayEntries = 5;
+
+/// Generates a fixed sized binary array for checking with a view.
+katana::Result<std::shared_ptr<arrow::FixedSizeBinaryArray>>
+GenerateTestArray() {
```

Comment:
I don't know this part of the code super well, but isn't there a file for testing property views already? Does this need to be in a separate file?

Reply:
I can place it in the older test file